### PR TITLE
Feat: LHCI Integration in CI workflow

### PR DESCRIPTION
**Pull request by dipanshuchaubey**
### Description

Adding LHCI benchmarks in CI workflow. It checks Lighthouse scores during CI.

---

### Checklist

- [x] PR description included
- [x] `yarn test` passes
- [ ] Tests are [changed or added]
- [ ] Relevant documentation is changed or added (and PR referenced)

### GIF's

Screenshot 2022-09-13 at 5 01 18 PM

---

### CODE REVIEW

File: internals/webpack/webpack.config.prod.js [hunk 1]
```diff
@@ -87,7 +87,7 @@ module.exports = require('./webpack.config.base')({
       AppCache: {
         events: true
       },
-      relativePaths: isProd() || isUAT(),
+      relativePaths: false,
```
Comment (ihatelactose) [inline internals/webpack/webpack.config.prod.js:90]: Why is this removed, shouldn't we be setting relative paths?

File: lighthouserc.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  ci: {
+    collect: {
+      url: ['http://localhost:3000/'],
```
Comment (ihatelactose) [inline lighthouserc.js:4]: This will only hit for `localhost`, is that right? Say if this was to run on a production will this still work?

Comment (dipanshuchaubey) [reply lighthouserc.js:4]: Yes, it will be running inside a ubuntu-latest container.

File: internals/webpack/webpack.config.prod.js (hunk 1 above)
Comment (ihatelactose) [reply internals/webpack/webpack.config.prod.js:90]: But completely getting rid of them would break if this was to be built on PROD or UAT, or would it not? Can you try building and see if everything is working or not?

File: lighthouserc.js (hunk 2 above)
Comment (ihatelactose) [reply lighthouserc.js:4]: Cool!

**Review by alichherawalla — DISMISSED** [review]

File: internals/webpack/webpack.config.prod.js (hunk 1 above)
Comment (alichherawalla) [inline internals/webpack/webpack.config.prod.js:90]: this will break the PR on UAT feature

Comment (dipanshuchaubey) [reply internals/webpack/webpack.config.prod.js:90]: Okay, I will revert this.

**Review by ihatelactose — APPROVED** [review]

File: package.json [hunk 3]
```diff
@@ -43,11 +43,14 @@
     "clean": "shjs ./internals/scripts/clean.js",
     "clean:all": "npm run analyze:clean && npm run test:clean && npm run build:clean",
     "lint": "npm run lint:js && npm run lint:css",
+    "lint:ci": "npm run lint:js:ci && npm run lint:css",
     "lint:css": "stylelint app/**/*.js",
     "lint:eslint": "eslint --ignore-path .eslintignore --ignore-pattern internals/scripts",
     "lint:eslint:fix": "eslint --ignore-path .eslintignore --ignore-pattern internals/scripts --fix",
     "lint:js": "npm run lint:eslint -- . ",
+    "lint:js:ci": "eslint --ignore-path .eslintignore internals/webpack/webpack.config.prod.js --ignore-pattern internals -- .",
```
Comment (ihatelactose) [inline package.json:51]: 🚀

**Review by alichherawalla — APPROVED** [review]
